frontend/integration-tests/tests/service-catalog/service-binding.scenario.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-undef, no-unused-vars */
+
+import {browser, $, $$, ExpectedConditions as until, by} from 'protractor';
+
+import { appHost, checkLogs, checkErrors, testName } from '../../protractor.conf';
+import * as srvCatalogView from '../../views/service-catalog.view';
+import * as sidenavView from '../../views/sidenav.view';
+import * as crudView from '../../views/crud.view';
+import {execSync} from "child_process";
+
+describe('Test for Cluster Service Binding', () => {
+  beforeAll(async() => {
+    browser.get(`${appHost}/status/ns/${testName}`);
+    await browser.wait(until.presenceOf($('#sidebar')));
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('creates a new binding for new service instance `mariadb-persistent`', async() => {
+    await sidenavView.clickNavLink(['Service Catalog', 'Service Classes']);
+    await srvCatalogView.isLoaded();
```

Comment:
We should be using `crudView.isLoaded()` here and below for any list / details page.